### PR TITLE
Use key-pair values on Vagrantfile

### LINKED ISSUE #116: Use key-pair values on Vagrantfile
Find a way to support .env files with Vagrant

### PR DESCRIPTION
**Pull request by Omeramsc**
fix #116 
Added .env file with Vagrant plugin

- As for now, there is nothing we can configure as an env variable, We'll have to accept #115 first.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -23,6 +26,7 @@ Vagrant.configure("2") do |config|
     stage.vm.box_url = "https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box"
 
     stage.vm.provider :aws do |aws,override|
+      override.nfs.functional = false 
       aws.keypair_name = "aws_key_pair"
```
Comment (lioramilbaum) [inline Vagrantfile:30]: aws_key_pair should not be hard coded

File: Vagrantfile [hunk 2]
```diff
@@ -23,6 +26,7 @@ Vagrant.configure("2") do |config|
...
       aws.ami = "ami-0701e7be9b2a77600"
```
Comment (lioramilbaum) [inline Vagrantfile:31]: same goes for ami

File: Vagrantfile [hunk 3]
```diff
@@ -23,6 +26,7 @@ Vagrant.configure("2") do |config|
     stage.vm.box_url = "https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box"
```
Comment (lioramilbaum) [inline Vagrantfile:26]: This line should be in the stage block

File: Vagrantfile [hunk 4]
```diff
@@ -10,6 +10,9 @@ class Hash
 end
 
 Vagrant.configure("2") do |config|
+  config.vagrant.plugins= ["vagrant-env"]
```
Comment (lioramilbaum) [inline Vagrantfile:13]: add vagrant-aws to the list.
BTW, have you tested that it works?

File: Vagrantfile [hunk 5]
```diff
@@ -23,6 +26,7 @@ Vagrant.configure("2") do |config|
     stage.vm.box_url = "https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box"
 
     stage.vm.provider :aws do |aws,override|
+      override.nfs.functional = false 
       aws.keypair_name = "aws_key_pair"
       aws.ami = "ami-0701e7be9b2a77600"
       aws.instance_type = "t2.micro"
```
Comment (lioramilbaum) [inline Vagrantfile:32]: same goes with instance_type

**Comment by Omeramsc** [comment]
@lioramilbaum Most of the comments you left are relevant to Ilan's PR and not things related to this effort, shouldn't it be changed on a separate PR?

**Comment by lioramilbaum** [comment]
@Omeramsc Fair enough

**Pull request merged**